Cargo.toml

```diff
@@ -19,3 +19,5 @@ path = "src/lib.rs"
 
 [dependencies]
 rand = "0.7"
+nalgebra = "0.19"
+num-traits = "0.2.10"
```

Comment:
do we still need the explicit dependency on `num-traits`?

Reply:
Nope, removed it.